metaflow/plugins/kubernetes/kube_utils.py

```diff
@@ -27,17 +27,19 @@ def parse_cli_options(flow_name, run_id, user, my_runs, echo):
 
 def qos_requests_and_limits(qos: str, cpu: int, memory: int, storage: int):
     "return resource requests and limits for the kubernetes pod based on the given QoS Class"
+    # case insensitive matching for QoS class
+    qos = qos.lower()
     # Determine the requests and limits to define chosen QoS class
     qos_limits = {}
     qos_requests = {}
-    if qos == "Guaranteed":
+    if qos == "guaranteed":
         # Guaranteed - has both cpu/memory limits. requests not required, as these will be inferred.
         qos_limits = {
             "cpu": str(cpu),
             "memory": "%sM" % str(memory),
             "ephemeral-storage": "%sM" % str(storage),
         }
-    elif qos == "BestEffort":
+    elif qos == "besteffort":
         # BestEffort - no limit or requests for cpu/memory
         qos_requests = {"ephemeral-storage": "%sM" % str(storage)}
```

Comment:
should we even support best effort at the moment? there doesn't seem to be a use case for it at the moment